packages/aesirx-dma-app/src/containers/ContentPage/ContentForm/ContentFormGeneral/index.jsx

```diff
@@ -158,19 +158,32 @@ const ContentFormGeneral = observer(
         });
       });
 
+      if (!this.isDisableHeadline()) {
+        if (
+          this.formPropsData[CONTENT_FIELD_KEY.NAME] === null ||
+          this.formPropsData[CONTENT_FIELD_KEY.NAME] === undefined
+        ) {
+          notify(t('txt_headline_is_null_or_undefined'), 'error');
```

Comment:
Please update translation for `txt_headline_is_null_or_undefined` 
Update in packages/aesirx-dma-app/src/translations if it only use for dma or update in Uikit if it use for multiple source